packages/pure-splitmix/pure-splitmix.0.3/opam

```diff
@@ -0,0 +1,16 @@
+opam-version: "2.0"
+maintainer: "Li-yao Xia <lysxia@gmail.com>"
+authors: "Li-yao Xia"
+homepage: "https://github.com/Lysxia/pure-splitmix"
+bug-reports: "https://github.com/Lysxia/pure-splitmix/issues"
+license: "MIT"
+dev-repo: "git+https://github.com/Lysxia/pure-splitmix.git"
+build: [ "dune" "build" "-p" name "-j" jobs ]
+depends: [
+  "dune" {build}
```

Comment:
```suggestion
  "dune" {>= "1.6"}
```
the build tag cannot be used due to a dune issue. see https://github.com/ocaml/dune/issues/2147